service-deployment/src/grafana/pai-clusterview-dashboard.json.template

```diff
@@ -579,7 +579,7 @@
           },
           "yaxes": [
             {
-              "format": "bytes",
+              "format": "bps",
```

Comment:
How about using "Bps"? It looks more familiar than "bps".

Reply:
Agree. 'b' usually refers to 'bit', 'B' is 'Byte'.

Reply:
If use "Bps", remember to remove "* 8" then.

Reply:
OK, changed to Bps. @hwuu @sterowang